store/mutations.js

```diff
@@ -7,6 +7,18 @@ export default {
     gql[attribute] = value
   },
 
+  setCollapsedSection({ theme }, value) {
+    theme.collapsedSections.includes(value)
+      ? (theme.collapsedSections = theme.collapsedSections.filter(section => section !== value))
+      : theme.collapsedSections.push(value)
+  },
+
+  setCollapsedSection({ theme }, value) {
+    theme.collapsedSections.includes(value)
+      ? (theme.collapsedSections = theme.collapsedSections.filter(section => section !== value))
+      : theme.collapsedSections.push(value)
+  },
```

Comment:
Duplicate code: This property is overwritten by another property in the same object literal.

https://lgtm.com/projects/g/liyasthomas/postwoman/rev/pr-b909ed91495c8b2ba56a610dcadf058045bea26b

Reply:
What the heck haha. Not sure how that duplicate got there. Fixed :)